### ISSUE TITLE
class="{css %Attribute}" produces syntax error: Unexpected token % in CSS tag

### ISSUE THREAD

**Issue by ryanseys** [Non-Dairy-Soy-Plugin/Non-Dairy-Soy-Plugin#32]
I noticed that this project started getting updates! Yay!

The `%` token is considered invalid in the parser but it is actually valid.

**Comment by SerCeMan** [comment]
Hi, @ryanseys!

For the awesome plugin all the "thanks" go to @evenaglia ! 

I'm not a really soy person, could you please give me a few examples of that?

**Comment by SerCeMan** [comment]
Hi, @ryanseys!

For the awesome plugin all the "thanks" go to @evenaglia ! 

I'm not a really soy person, could you please give me a few examples of that?

**Comment by ryanseys** [comment]
Basically the `%` token can be an optional first character in the css name.

#### valid.soy

```html
<div class="{css %Container}"></div>
```

#### notvalid.soy

```html
<div class="{css Cont%ainer}"></div>
```

**Comment by ryanseys** [comment]
Might be as simple as updating this: https://github.com/Non-Dairy-Soy-Plugin/Non-Dairy-Soy-Plugin/blob/9e4150171f445307c6652b4317d54ba39d8ef5e5/src/main/grammars/_SoyLexer.flex#L267

**Comment by SerCeMan** [comment]
> Might be as simple as updating this

Not exactly, `%` gets parsed as `MOD`, I put a check into the parser https://github.com/Non-Dairy-Soy-Plugin/Non-Dairy-Soy-Plugin/commit/861df4378f49582a7dd00ab7bca4515ba51f209c

**Comment by SerCeMan** [comment]
Hi, @ryanseys!

The issue has been fixed with the new release [0.7.1](https://github.com/Non-Dairy-Soy-Plugin/Non-Dairy-Soy-Plugin/releases/tag/0.7.1). Feel free to reopen the issue if the problem still exists for you.

**Comment by ryanseys** [comment]
Thanks!!
